docs/meta/supported-os.md

```diff
@@ -6,31 +6,30 @@ nav_order: 2
 has_children: false
 parent: Information
 search_exclude: false
-last_modified_date: 2023-02-27
+last_modified_date: 2023-12-25
 ---
 
 # Supported Operating Systems
 
-## Full Support:
-* Non-EOL, consumer versions of Windows 10/11 only:
+## Fully Supported
+* All actively maintained versions of Windows 10 and Windows 11, provided they are not end-of-life (EOL) and are consumer editions:
     * Home
     * Pro
+    * Pro Education
     * Pro for Workstations
-    * Education
 
-## No Support:
-* Any EOL OS, including but not limited to:
+## Not Supported
+* Any operating system that has reached end-of-life (EOL), including but not limited to:
     * Windows XP
     * Windows Vista
     * Windows 7
     * Windows 8/8.1
-    * Windows 10/11 versions that have lost support [(1507-21H1 at time of writing)](https://docs.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro)
-    * Fixed-release Linux distro versions, consult your distro's website
-    * Rolling-release Linux installations that are not fully up-to-date (Arch-based distros are usually rolling-release)
-<!--    * Standard versions of Ubuntu 21.04 and older, and LTS versions of Ubuntu 12.04 and older -->
-* Enterprise/LTSC/LTSB versions of Windows
-* Hackintosh
-* *Any* modified image of Windows (e.g. Tiny11, AtlasOS, etc.)
+    * Certain versions of Windows 10/11 (e.g., Windows 10 versions 1507 to 21H2; for more details, visit the [Windows lifecycle fact sheet](https://learn.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro))
```

Comment:
If we're specifically referring to versions of Windows that went EOL, I'd write it as:
```
* Older versions of Windows 10/11 that have lost support (Windows 10 versions 1507 to 21H2, and Windows 11 21H2 as of update date; for more details, visit the Lifecycle Fact Sheet for [Windows 10](https://learn.microsoft.com/en-us/lifecycle/products/windows-10-home-and-pro) and [Windows 11](https://learn.microsoft.com/en-us/lifecycle/products/windows-11-home-and-pro))
```